### ISSUE TITLE
Installing ember-cli-deploy on ember-cli@1.13.0 fails

### ISSUE THREAD

**Issue by tim-evans** [ember-cli/ember-cli#4397]
Steps to reproduce:

1. ember new my-app
2. ember install ember-cli-deploy
3. Boom.

```
Cannot find module 'ember-cli/lib/errors/silent'
Error: Cannot find module 'ember-cli/lib/errors/silent'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/Users/user/my-app/node_modules/ember-cli-deploy/lib/utilities/configuration-reader.js:4:19)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
```

**Comment by tim-evans** [comment]
Resolved by https://github.com/ember-cli/ember-cli-deploy/pull/176

**Comment by rwjblue** [comment]
We need to bring back `lib/errors/silent` that is deprecated, but returns `silent-error`.

**Comment by tim-evans** [comment]
Thanks @rwjblue!!
Always on top of things :3

**Comment by stefanpenner** [comment]
> We need to bring back lib/errors/silent that is deprecated, but returns silent-error, and release 1.13.1.

Should be easy, good call.

**Comment by trabus** [comment]
@tim-evans just released v1.13.1, please let me know if you are still having issues. You should see a deprecation warning when it requires `ember-cli/lib/errors/silent`.
